### DISCUSSION TITLE
loss curve

### DISCUSSION THREAD

**Discussion by yangrui12321** [NVlabs/instant-ngp#550] [Q&A]
How can I get the loss curve during training? Where can this data be exported? Thank you so much!

**Comment by Tom94** [comment]
You can use the Python bindings to query `testbed.loss` inbetween training steps. In general, check `python_api.cu` for all the available bindings.

**Reply by afiretony** [reply]
Hi, can I set loss type in python bingings? Thanks!
